crm_project_task/tests/test_crm_project_task.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+from odoo.exceptions import UserError
+from odoo.tests import tagged
+from odoo.tests.common import TransactionCase
+
+from odoo.addons.mail.tests.common import mail_new_test_user
+
+
+@tagged("post_install", "-at_install")
+class TestCrmProjectTask(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.company = cls.env.user.company_id
+        cls.user_salesman = mail_new_test_user(
+            cls.env,
+            login="user_test",
+            name="User Test",
+            email="user_test@test.example.com",
+            company_id=cls.company.id,
+            groups="sales_team.group_sale_salesman",
+        )
+        cls.partner = cls.env["res.partner"].create(
+            {
+                "name": "Partner Test",
+            }
+        )
+        cls.lead = cls.env["crm.lead"].create(
+            {
+                "name": "Test Lead",
+                "type": "lead",
+                "partner_id": cls.partner.id,
+                "user_id": cls.user_salesman.id,
+            }
+        )
+        cls.project = cls.env["project.project"].create(
+            {
+                "name": "Test Project",
+                "description": "Test Description",
+            }
+        )
+
+    def test_create_task(self):
+        self.company.crm_default_project_id = self.project
+        self.task_name = "Task Test"
+        self.task_description = "Line1</br>Line2"
```

Comment:
As a rule of thumb, we should avoid creating new attributes on `self` in unit tests.

The reason is that, if we forget about them, they will be kept for next tests. This can introduce unwanted side effects.

So, generally, all attributes are created in the setup stage.

```suggestion
        company.crm_default_project_id = self.project
        task_name = "Task Test"
        task_description = "Line1</br>Line2"
```

Reply:
Applied.